modules/examples/src/main/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementExamples.java

```diff
@@ -0,0 +1,357 @@
+          .primaryContactIamId(enterpriseAccountIamId)
+          .build();
+
+      Response<CreateAccountGroupResponse> response = service
+          .createAccountGroup(createAccountGroupOptions)
+          .execute();
+      CreateAccountGroupResponse createAccountGroupResponse = response.getResult();
+
+      System.out.println(createAccountGroupResponse);
+      // end-create_account_group
+      secondAccountGroupId = createAccountGroupResponse.getAccountGroupId();
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-list_account_groups
+      ListAccountGroupsOptions listAccountGroupsOptions = new ListAccountGroupsOptions.Builder()
+          .enterpriseId(enterpriseId)
+          .build();
+
+      Response<ListAccountGroupsResponse> response = service
+          .listAccountGroups(listAccountGroupsOptions)
+          .execute();
+      ListAccountGroupsResponse listAccountGroupsResponse = response.getResult();
+
+      System.out.println(listAccountGroupsResponse);
+      // end-list_account_groups
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-get_account_group
+      GetAccountGroupOptions getAccountGroupOptions = new GetAccountGroupOptions.Builder()
+          .accountGroupId(firstAccountGroupId)
+          .build();
+
+      Response<AccountGroup> response = service
+          .getAccountGroup(getAccountGroupOptions)
+          .execute();
+      AccountGroup accountGroup = response.getResult();
+
+      System.out.println(accountGroup);
+      // end-get_account_group
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-update_account_group
+      UpdateAccountGroupOptions updateAccountGroupOptions = new UpdateAccountGroupOptions.Builder()
+          .accountGroupId(firstAccountGroupId)
+          .name(firstUpdatedAccountGroupName)
+          .build();
+
+      service.updateAccountGroup(updateAccountGroupOptions)
+          .execute();
+      // end-update_account_group
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-import_account_to_enterprise
+      ImportAccountToEnterpriseOptions importAccountToEnterpriseOptions = new ImportAccountToEnterpriseOptions.Builder()
+          .enterpriseId(enterpriseId)
+          .accountId("Standalone Account Id")
```

Comment:
What does "standalone" mean in this context?  Also, shouldn't this be some sort of "id" value?

Reply:
You are right, it is misleading. The documentation says below. In my opinion `Existing Account Id` would be better. The Id of the account in this case is string, it is a GUID/UUID.

```Import an existing stand-alone account into an enterprise. The existing account can be any type: trial (TRIAL), Lite (STANDARD), Pay-As-You-Go (PAYG), or Subscription (SUBSCRIPTION).```